### DISCUSSION TITLE
Sonraki bölüme geçme

### DISCUSSION THREAD

**Discussion by ahmehmet** [KebabLord/turkanime-indirici#30] [Genel]
Her bölümü bitirdikten sonra tekrar tekrar anime aratmak yerine diğer bölüme direkt geçme tuşu ekleyebilir misiniz?

**Comment by KebabLord** [comment]
Son sürümle birlikte artık seçim ekranı son izlenilen bölümden başlayacak, exe olarak da indirilebilir.
[video](https://user-images.githubusercontent.com/24779257/209420172-630e3e8d-a592-42e6-86ba-bd2b57a6408d.webm)
